operator/roles/forkliftcontroller/templates/monitor/recordingrole-migrations.yml.j2

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
```

Comment:
Q: do we need labels to identify this rule as belonging to the forklift operator application ?

Reply:
The name of the recording rule " cluster:mtv_migrations_status_total:max" includes the mtv.
I think it should be sufficient.